### DISCUSSION TITLE
Block coding for EV3

### DISCUSSION THREAD

**Discussion by techcatgato** [pybricks/support#1672] [Q&A]
Just to clarify, Is there really no way to use block coding or modern pybricks on EV3? (Above 2.0)

**Comment by laurensvalk** [comment]
I'm afraid not.

We'd like to get there eventually though :)

**Reply by techcatgato** [reply]
Not even 3.0? I saw some posts talking about it

**Reply by laurensvalk** [reply]
Nothing that we can recommend, no.
